.github/workflows/ci-sharded.yml

```
@@ -0,0 +1,331 @@
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: comic_pile_test
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        command: postgres -c max_connections=200
+    steps:
+      - name: Check out repository
```

actionlint (1.7.10)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)